docs/pages/_app.js

```diff
@@ -1,3 +1,11 @@
+/* eslint-disable import/first */
+import { LicenseInfo } from '@material-ui/x-grid';
+
+// Remove the license warning from demonstration purposes
+LicenseInfo.setLicenseKey(
+  '0f94d8b65161817ca5d7f7af8ac2f042T1JERVI6TVVJLVN0b3J5Ym9vayxFWFBJUlk9MTY1NDg1ODc1MzU1MCxLRVlWRVJTSU9OPTE=',
+);
```

Comment:
Let's not document it that way. License keys have no place in checked in code. I would expect that this doesn't work that way in the future. Remix has a simpler approach that doesn't require any runtime work.

Reply:
I follow this pattern from x repo docs. @DanailH @m4theushw do we already have a plan or discussed about the approach?

Reply:
This means that the license key leaks into the deployed bundle unless there's a transpiler/bundler plugin that strips it.

Reply:
I think that we should move this discussion to #27663 as this PR is based on it.

Reply:
@oliviertassinari if we need to discuss it, I will not include `XGrid` in homepage for now (use DataGrid instead) because I feel that the discussion will take time and I don't want to wait on #27663.